electron/utils.ts

```diff
@@ -208,11 +208,18 @@ async function checkCommandVersion(
   return found;
 }
 
+async function fetchFreeProducts() {
```

Comment:
Bring over all functions from FreeProducts/utils.tsx and prepare the data here.
The Return should be Element[].